### PR TITLE
[BugFix] Update Captions banner to not obstruct verticalGallery controls on landscape mobile

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

add compact mode to captions component to allow more vertical space on mobile.
moves control bar to left side when on landscape mobile 
image
image
image

# Why

Allows better experience for captions and overflow gallery on landscape mobile

https://skype.visualstudio.com/SPOOL/_workitems/edit/3217746
# How Tested

locally see images ^ will update snapshots

### CODE REVIEW

File: change/@azure-communication-react-20b35cc7-52f0-40ed-8502-191c5fe29c38.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Update Captions component for mobile landscape to not obstruct the verticalGallery",
```
Comment (edwardlee-msft) [inline change/@azure-communication-react-20b35cc7-52f0-40ed-8502-191c5fe29c38.json:3]:
```suggestion
  "comment": "Update Captions component for mobile landscape to not obstruct the VerticalGallery",
```

File: packages/react-components/src/components/CaptionsBanner.tsx [hunk 2]
```diff
@@ -47,14 +47,19 @@ export interface _CaptionsBannerProps {
    */
   onRenderAvatar?: OnRenderAvatarCallback;
   strings?: _CaptionsBannerStrings;
+  /**
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/CaptionsBanner.tsx:50]: so this is a nit, but can you fix line 29 comment... seems like banner is spelled with an extra e.

File: packages/react-composites/src/composites/common/CaptionsBanner.tsx [hunk 3]
```diff
@@ -73,19 +68,25 @@ export const CaptionsBanner = (props: { isMobile: boolean }): JSX.Element => {
...
-              <_CaptionsBanner {...captionsBannerProps} {...handlers} strings={captionsBannerStrings} />
-            </Stack.Item>
-          </Stack>
-          {!props.isMobile && captionsBannerProps.isCaptionsOn && (
-            <div className={floatingChildClassName}>
-              <CaptionsBannerMoreButton onCaptionsSettingsClick={onClickCaptionsSettings} />
-            </div>
-          )}
-        </div>
+        /* @conditional-compile-remove(close-captions) */ captionsBannerProps.isCaptionsOn && (
```
Comment (carocao-msft) [inline packages/react-composites/src/composites/common/CaptionsBanner.tsx:71]: Why do we add a check here? We still wants captions banner to show when captions is not started yet but user already clicked start captions (loading banner)

**Comment by edwardlee-msft** [comment]
nit: would it be possible to not impact portrait mobile captions component height size?

**Review by edwardlee-msft — APPROVED** [review]

File: change-beta/@azure-communication-react-20b35cc7-52f0-40ed-8502-191c5fe29c38.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Update Captions component for mobile landscape to not obstruct the verticalGallery",
```
Comment (edwardlee-msft) [inline change-beta/@azure-communication-react-20b35cc7-52f0-40ed-8502-191c5fe29c38.json:3]:
```suggestion
  "comment": "Update Captions component for mobile landscape to not obstruct the VerticalGallery",
```

**Review by edwardlee-msft — APPROVED** [review]

File: packages/react-composites/src/composites/common/CaptionsBanner.tsx [hunk 5]
```diff
@@ -73,11 +68,15 @@ export const CaptionsBanner = (props: { isMobile: boolean }): JSX.Element => {
         )
       }
       {
-        /* @conditional-compile-remove(close-captions) */
-        <div className={containerClassName}>
+        /* @conditional-compile-remove(close-captions) */ <div className={containerClassName}>
           <Stack horizontalAlign="center">
-            <Stack.Item style={{ width: props.isMobile ? mobileViewBannerWidth : desktopViewBannerWidth }}>
-              <_CaptionsBanner {...captionsBannerProps} {...handlers} strings={captionsBannerStrings} />
+            <Stack.Item style={{ width: props.isMobile ? '90%' : '50%' }}>
```
Comment (carocao-msft) [inline packages/react-composites/src/composites/common/CaptionsBanner.tsx:73]: Should we use the mobileViewBannerWidth : desktopViewBannerWidth here?

**Review by carocao-msft — APPROVED** [review]

**Pull request merged**